src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -60,7 +68,7 @@ public void takesHeadOutOfHttpResponse() throws IOException {
                     )
                 )
             ).asString(),
-            Matchers.endsWith("text/plain")
```

Comment:
@ilyakharlamov please replace this with cactoos-matchers `EndsWith` (and don't call `asString()` on the `Text`)